config/config.go

```diff
@@ -598,13 +597,6 @@ func validatePipeline(cfg *configmodels.Config, pipeline *configmodels.Pipeline)
 }
 
 func validatePipelineReceivers(cfg *configmodels.Config, pipeline *configmodels.Pipeline) error {
-	if len(pipeline.Receivers) == 0 {
```

Comment:
While this might be required for your use case, the overwhelming majority still need a receiver, so I would prefer to keep the default behavior requiring at least one receiver.

Shall we add some flag to the pipeline config to explicitly enable your "receiverless" mode?